packages/block-library/src/post-excerpt/block.json

```diff
@@ -1,4 +1,17 @@
 {
 	"name": "core/post-excerpt",
-	"category": "layout"
+	"category": "layout",
+	"attributes": {
+		"excerptLength": {
+			"type": "number",
+			"default": 55
+		},
+		"moreText": {
+			"type": "string"
+		},
+		"showMoreOnNewLine": {
```

Comment:
Should this be an attribute or just something a theme can tweak with CSS?

Reply:
Making it an attribute allows us to leverage different markup.